torch/_torch_docs.py

```diff
@@ -6648,9 +6648,8 @@ def merge_dicts(*dicts):
     The difference between ``max``/``min`` and ``amax``/``amin`` is:
         - ``amax``/``amin`` supports reducing on multiple dimensions,
         - ``amax``/``amin`` does not return indices,
-        - ``amax``/``amin`` evenly distributes gradient between equal values,
-          while ``max(dim)``/``min(dim)`` propagates gradient only to a single
-          index in the source tensor.
+        - ``amax``, ``amin``, ``max(dim)``, ``min(dim)` evenly distributes gradient between equal values 
```

Comment:
Could you reword this to:
`Both ``max``/``min`` and ``amax``/``amin` evenly distribute gradients between equal values`

Also we should unindent since so we're no longer a bullet point in the differences section